prism/src/main/java/org/apache/falcon/jdbc/MonitoringJdbcStateStore.java

```diff
@@ -48,7 +48,7 @@ public void putMonitoredEntity(String entityName, String entityType) throws Falc
 
         MonitoredEntityBean monitoredEntityBean = new MonitoredEntityBean();
         monitoredEntityBean.setEntityName(entityName);
-        monitoredEntityBean.setEntityType(entityType);
+        monitoredEntityBean.setEntityType(entityType.toLowerCase());
```

Comment:
Then, lets change it in the Bean itself as that is closest to the DB and the conversation will need to be done in lesser number of places.